save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/UsersDetailsController.kt

```diff
@@ -173,12 +172,13 @@ class UsersDetailsController(
 
     /**
      * @param authentication
-     * @return global [Role] of authenticated user
+     * @return [UserInfo] of authenticated user
      */
-    @GetMapping("/global-role")
+    @GetMapping("/user-info")
     @PreAuthorize("isAuthenticated()")
-    fun getSelfGlobalRole(authentication: Authentication): Mono<Role> =
-            Mono.just(userDetailsService.getGlobalRole(authentication))
+    fun getSelfUserInfo(authentication: Authentication): Mono<UserInfo> = blockingToMono {
```

Comment:
probably better to accept here `Authentication?` and return empty `Mono` to avoid 401 as currently for `global-role`

Reply:
done, added this endpoint to public endpoints too